codegen/src/main/stubs/org/nd4j/linalg/api/buffer/DataType.java

```diff
@@ -0,0 +1,28 @@
+package org.nd4j.linalg.api.buffer;
+
+public enum DataType {
```

Comment:
can you stub this out?

I want to make sure that stubs only have the minimum amount of things in there,  so if we ever have to change things, we know exactly what we are actually directly referencing

Reply:
Datatype enums values are required, at least for BOOL and UTF8
https://github.com/KonduitAI/dl4j-dev-tools/blob/626ec1e1348fbc64566b3bf6f9a46ee3148850a9/codegen/src/main/resources/java/utilClasses/NDValidation.java#L40

Same thing for isIntType and isFPType

One option is to simply stub NDValidation here, have the real one in ND4J, and never copy back... I'd be fine with that instead.

Reply:
For the record:
We discussed this, and NDValidation is meant to be stubbed here, and not copied into ND4J. The copying mechanism was just a crutch while nd4j didn't have NDValidation.